tests/test_wave_01.py

```diff
@@ -143,12 +142,11 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
     # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
+    assert updated_data["watched"][1]["title"] == movie_to_watch["title"]
```

Comment:
Nice assertion! You could also add assertions for checking which movie is remaining in the watchlist list, and another for the other movie we expect to find in the watched list.